scripts/a11y/index.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs');
+const util = require('util');
+const puppeteer = require('puppeteer');
+const axe = require('axe-core');
+const axeReports = require('axe-reports');
+
+const fileContents = fs.readFileSync('./dev-site-config/build/routeConfig.js').toString().split('export default');
+
+const paths = [
+  'https://engineering.cerner.com/terra-ui/#/home/terra-ui/index',
+  'https://engineering.cerner.com/terra-ui/#/getting-started/terra-ui/what-is-terra',
+];
+
+const urls = [];
+const prefixURL = 'https://engineering.cerner.com/terra-ui/#'
+
+var stringContents = fileContents[1];
+
+var words = stringContents.split(/'path': '/);
+
+// add all the URLs from routeConfig.js to an array
+for(i = 0; i < words.length; i++) {
+  var url = words[i]
+  var n = url.indexOf(/'/);
+  url = url.substring(0, url.indexOf("'"));
+  url = prefixURL + url;
+  urls.push(url)
+};
+
+urls.shift(); //remove faulty firsty entry '{\n   }'
+
+urls.push(paths[0]);  // add 2 urls not included in routeConfig.js
+urls.push(paths[1]);
+
+const results = [];
+let axeResults;
+let promise;
+const time = process.hrtime();
+puppeteer.launch().then(async (browser) => {
+  const promises = [];
+  console.log("Running...");
+  for (let i = 0; i < urls.length; i++) {
+    const url = urls[i];
+    promise = await browser.newPage().then(async (page) => {
+      await page.goto(`${url}`, {
+                waitUntil: 'load',
+            });
+      // add axe-core to the pages
+      await page.addScriptTag({
+        path: require.resolve('axe-core')
+      });
+      console.log('[' + i + '/' + urls.length-1 + ']   ', url);
+      // run axe on the page
+      axeResults = await page.evaluate(async () => {
+        return await axe.run(document, {
+          runOnly: ['wcag2aa', 'section508'],
+          resultTypes: ['violations']
```

Comment:
do we want to limit the reports to violations?

Reply:
Yeah, I believe so. I don't think the other reports are valuable in this context.